types_test.go

```diff
@@ -129,3 +129,47 @@ func TestToSignedTreeHead(t *testing.T) {
 		})
 	}
 }
+
+func TestSTHString(t *testing.T) {
+	tests := []struct {
+		desc      string
+		logID     string
+		wantLogID bool
+	}{
+		{
+			desc:      "no logID",
+			wantLogID: false,
```

Comment:
Maybe just use `len(test.logID) != 0`, and drop `wantLogID`?

Reply:
Done.